src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
```

Comment:
Consider defining constants for the number of days that qualify for a long-term and short-term discount, instead of using hard-coded values '7' and '3' in the if conditions (lines 9 and 13), as per the checklist item 2.